main/src/ca/uwaterloo/flix/language/dbg/printer/TypedAstPrinter.scala

```diff
@@ -76,6 +76,7 @@ object TypedAstPrinter {
     case Expr.TryCatch(exp, rules, _, _, _) => DocAst.Expression.TryCatch(print(exp), rules.map(printCatchRule))
     case Expr.TryWith(exp, _Use, rules, _, _, _) => DocAst.Expression.TryWith(print(exp), _Use.sym, rules.map(printHandlerRule))
     case Expr.Do(op, exps, _, _, _) => DocAst.Expression.Do(op.sym, exps.map(print))
+    case Expr.InvokeMethod2(exp, name, exps, _, _, _) => DocAst.Expression.JavaInvokeMethod2(name, print(exp), exps.map(print)) // TO CHECK
```

Comment:
This seems reasonable to me

Reply:
Except, why do we switch the order of parameters?

Reply:
It should be changed.